app/src/main/res/layout/dialog_safe_selection.xml

```diff
@@ -36,6 +36,7 @@
             android:layout_width="match_parent"
             android:layout_height="0dp"
             android:layout_marginTop="16dp"
+            android:nestedScrollingEnabled="true"
```

Comment:
The point of my previous comment was to set this to `false`, because the recycler_view is contained already in a nested scroll view.